### PR TITLE
add Git Beginner, Git remote whitelist/blacklist, Git Travis CI project

### PR DESCRIPTION
**Pull request by larsxschneider**

### CODE REVIEW

File: SoC-2016-Microprojects.md [hunk 1]
```diff
@@ -173,3 +173,104 @@ It should only show the error message instead. Cf. $gmane/284328
...
+efficiently with Git (e.g. by maintaining a nice history using 
+`rebase`). However, these users are not the target audience. The target
+audience are beginners that want or need to use Git and want to
+stay out of trouble.
+
+### Git remote whitelist/blacklist
+
+Git beginners are easily confused by the distributed nature of Git.
+One source of confusion are Git remotes, especially if there are 
+multiple ones. This is a potentially big thread to cooperations
```
Comment (pranitbauva1997) [inline SoC-2016-Microprojects.md]: potentially big threat to cooperations

Comment (larsxschneider) [inline SoC-2016-Microprojects.md]: fixed. thanks :)

**Comment by larsxschneider** [comment]
@moy Updated according to your suggestions! The commits are a bit messy. Let me know if you prefer a clean history in this repo then I will rebase them. :smile:

**Comment by moy** [comment]
I slightly prefer a clean history, but this page is intermediate between a clean repo and a wiki so I don't mind too much. I had to fix a merge conflict anyway, so I did a `merge --squash`.

**Pull request closed without merging**